openreview/venue_request/venue_request.py

```diff
@@ -1616,6 +1616,19 @@ def setup_request_form(self):
                 'description': 'Please briefly describe how you heard about OpenReview.',
                 'order': 41
             },
+            'venue_organizer_agreement': {
+                'description': 'In order to use OpenReview, venue chairs must agree to the following:',
+                'values-checkbox': [
+                    'OpenReview natively supports a wide variety of reviewing workflow configurations. However, if we want significant reviewing process customizations or experiments, we will detail these requests to the OpenReview staff at least three months in advance.',
+                    'We will ask authors and reviewers to create an OpenReview Profile at least two weeks in advance of the paper submission deadlines.',
+                    'When assembling our group of reviewers and meta-reviewers, we will only include email addresses or OpenReview Profile IDs of people we know to have authored publications relevant to our venue.  (We will not solicit new reviewers using an open web form, because unfortunately some malicious actors sometimes try to create \'fake ids\' aiming to be assigned to review their own paper submissions.)',
```

Comment:
@xkopenreview You can use this PR to check the issue I mentioned. I am using single-quotes here and escaping them, since using double-quotes does not work.